commons/src/main/java/com/ge/predix/acs/commons/policy/condition/AbstractHandler.java

```diff
@@ -79,7 +79,7 @@ public String toString() {
         return "AbstractHandler [attributeMap=" + this.attributeTypeMap + "]";
     }
 
-    public AbstractHandler has(final AttributeType attributeType) throws ConditionAssertionFailedException {
+    public AbstractHandler has(final AttributeType attributeType) {
```

Comment:
There are lots of methods that we throw Runtime Exceptions and don't declare it in the method signature. I sort of feel like it adds noise but doesn't have benefit because the caller isn't forced to handle.

Do you think that if we feel the caller should handle this exception, it should be a checked exception?